pcsx2/windows/VCprojects/IopSif.cpp

```diff
@@ -90,12 +90,6 @@ void FinalizeEERead()
 	CPU_INT(DMAC_SIF0, 16);
 }
 
-s32 DoSIFWrite(u32 iopAvailable)
-{
-	u32 eeAvailable = PrepareEEWrite();
```

Comment:
This is undefined but probably returning eeAvailable on accident. So the code in sif0DmaRead was probably doing what it was intended to do. Or is the code dead for other reasons?

Reply:
The whole file isn't used, I'll be purging it.

Reply:
Why bother making modifications to a file you're about to delete?

Maybe just delete it now

Reply:
That's what I said?